Make gen_changelog.sh not do anything since it doesn't work unless it's
run in a git clone.

https://github.com/BestImageViewer/geeqie/issues/568
--- gen_changelog.sh.orig	2017-12-31 06:31:21.000000000 -0600
+++ gen_changelog.sh	2019-06-23 02:56:24.000000000 -0500
@@ -7,6 +7,8 @@
 #
 # ChangeLog.html is also created
 
+touch ChangeLog ChangeLog.html
+exit 0
 [ ! -e "ChangeLog.gqview" ] && exit 1
 [ ! -x "$(command -v git)" ] && exit 0
 
